kf_lib_data_ingest/target_apis/kids_first.py

```diff
@@ -269,3 +287,143 @@
         CONCEPT.SEQUENCING
     }
 }
+
+
+def _get_kidsfirst_schema(url, cached_schema_filepath=CACHED_SCHEMA_FILE):
```

Comment:
Most of this function doesn't seem specific to the kids first dataservice. It seems more like an openapi schema consumer?

Reply:
Same as above, wasn't sure about this. The only thing that is Kids First Data Service specific is getting the version information from the `/status` endpoint. That might not exist for every service or if it does the response content could be different

Reply:
version at least is included in the schema endpoint. I think just the commit shorthash isn't there

Reply:
Ah perfect, that's probably good enough right?

Reply:
I suppose that depends on whether we trust the dataservice to bump version numbers properly